data/deprecated.json

```diff
@@ -175,6 +175,22 @@
     "old": {"amenity": "vending_machine", "vending": "news_papers"},
     "replace": {"amenity": "vending_machine", "vending": "newspapers"}
   },
+  {
+    "old": {"amenity": "vending_machine", "vending": "parcel_pickup;parcel_mail_in"},
+    "replace": {"amenity": "parcel_locker", "parcel_mail_in": "yes"}
+  },
+  {
+    "old": {"amenity": "vending_machine", "vending": "parcel_mail_in"},
+    "replace": {"amenity": "parcel_locker", "parcel_mail_in": "yes"}
```

Comment:
shouldn't this one have `"parcel_pickup": "no"` ?

Reply:
You're right. Fixed it.